contracts/evoting/evoting.go

```diff
@@ -65,6 +65,20 @@ func (e evotingCommand) createForm(snap store.Snapshot, step execution.Step) err
 		return xerrors.Errorf("failed to get roster")
 	}
 
+	// Check if has Admin Right to create a form
+	adminList, err := e.getAdminList(snap)
+	if err != nil {
+		return xerrors.Errorf("failed to get AdminList: %v", err)
+	}
+	adminIndex, err := adminList.GetAdminIndex(tx.UserID)
+	if err != nil {
+		return xerrors.Errorf("Couldn't retrieve the admin right of the user: %v", err)
+	}
+
+	if adminIndex < 0 {
+		return xerrors.Errorf("The user is not admin: %v", err)
+	}
```

Comment:
My problem by moving this part to the is_admin is that it is used to check whether the Admin List already exist and in case it does not, create the Admin List (initialize admin list procedure) -> Therefore I would find weird that it is the is_admin that perform the logic of initializing the Admin List.

What is your opinion about it ?